src/components/EditImageModal/EditImageModal.test.jsx

```diff
@@ -30,7 +30,7 @@ const sampleImgData = {
 };
 sampleImgData.aspectRatio = sampleImgData.naturalWidth / sampleImgData.naturalHeight;
 
-const updateConsants = () => {
```

Comment:
I think this name was typo-ed so I did a find-and-replace to fix it in this file.

Reply:
My bad. Thank you.